com.valvesoftware.SteamLink.yml

```
@@ -0,0 +1,208 @@
+      - /lib/*.prl
+      - /lib/cmake
+      - /lib/debug
+      - /lib/mkspecs
+      - /lib/pkgconfig
+
+  - name: qt5-qtsvg
+    buildsystem: qmake
+    cleanup:
+      - /bin
+      - /include
+      - /lib/*.a
+      - /lib/*.la
+      - /lib/*.prl
+      - /lib/cmake
+      - /lib/debug
+      - /lib/mkspecs
+      - /lib/pkgconfig
+    sources:
+      - type: archive
+        url: http://download.qt.io/archive/qt/5.14/5.14.1/submodules/qtsvg-everywhere-src-5.14.1.tar.xz
+        mirror-urls:
+          - https://repo.steampowered.com/hidden/steamlink/8540a57312f815f81a45b891b49959d776727fde17579bb6bf1a537996bc9359/qtsvg-everywhere-src-5.14.1.tar.xz
+        sha256: 8540a57312f815f81a45b891b49959d776727fde17579bb6bf1a537996bc9359
+
+  - name: extra-cmake-modules
+    buildsystem: cmake-ninja
+    cleanup:
+      - '*'
+    sources:
+      - type: archive
+        url: https://download.kde.org/stable/frameworks/5.78/extra-cmake-modules-5.78.0.tar.xz
+        mirror-urls:
+          - https://repo.steampowered.com/hidden/steamlink/eede1f0c21f24eca56bd71a805fa76500a8c683bf21982cb33f4bba682d012f9/extra-cmake-modules-5.78.0.tar.xz
+        sha256: eede1f0c21f24eca56bd71a805fa76500a8c683bf21982cb33f4bba682d012f9
+      # also available as:
+      # type: git
+      # url: https://invent.kde.org/frameworks/extra-cmake-modules.git
+      # tag: v5.78.0
+      # commit: 59d4958740a09d43aa5025a1cfe3d7b7d4d725b3
+
+  - name: kdbusaddons
+    buildsystem: cmake-ninja
+    cleanup:
+      - '/bin'
+      - '/include'
+      - '/lib/cmake'
+      - '/lib/mkspecs'
+    config-opts:
+      - -DENABLE_TESTING=OFF
+    sources:
+      - type: archive
+        url: https://download.kde.org/stable/frameworks/5.78/kdbusaddons-5.78.0.tar.xz
+        mirror-urls:
+          - https://repo.steampowered.com/hidden/steamlink/4f91d0b5762243e5e044c087d4faaa584ad5f779fdede138a0048fd4af24d399/kdbusaddons-5.78.0.tar.xz
+        sha256: 4f91d0b5762243e5e044c087d4faaa584ad5f779fdede138a0048fd4af24d399
+      # also available as:
+      # type: git
+      # url: https://invent.kde.org/frameworks/kdbusaddons.git
+      # tag: v5.78.0
+      # commit: e70c696143682bf23509aed4c31d5fc95093d452
+
+      # if we delete the l10n (which we don't want anyway) then it drops the
+      # dependency on QtLinguist
+      - type: shell
+        commands:
+          - rm -fr po
+
+  - name: steamlink-binary
+    sources:
+      - type: archive
+        url: https://repo.steampowered.com/hidden/steamlink/b8b1b94651e7a4cfd6335180697cd35dda771058f942900b5775a69720d218c8/steamlink-1.1.73.179.tgz
+        sha256: b8b1b94651e7a4cfd6335180697cd35dda771058f942900b5775a69720d218c8
+        strip-components: 1
+    buildsystem: simple
+    build-commands:
+      - "install bin/steamlink /app/bin"
+      - "cp -a lib/* /app/lib"
+      - "cp -a *.txt ThirdPartyLegalNotices.* /app"
+
+  - name: packaging
+    sources:
+      - type: file
+        path: com.valvesoftware.SteamLink.appdata.xml
```

```suggestion
        path: com.valvesoftware.SteamLink.metainfo.xml
```
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html